src/dbnode/persist/fs/types.go

```diff
@@ -244,9 +244,9 @@ type DataFileSetSeekerManager interface {
 	// Return returns an open seeker for a given shard, block start time, and volume.
 	Return(shard uint32, start time.Time, seeker ConcurrentDataFileSetSeeker) error
 
-	// ConcurrentIDBloomFilter returns a concurrent ID bloom filter for a given
-	// shard, block start time, and volume.
-	ConcurrentIDBloomFilter(shard uint32, start time.Time) (*ManagedConcurrentBloomFilter, error)
+	// Test checks if an ID exists in a concurrent ID bloom filter for a
+	// given shard, block, start time and volume.
+	Test(shard uint32, id ident.ID, start time.Time) (bool, error)
```

Comment:
super nit: Perhaps we make this `Test(id ident.ID, shard uint32, start time.Time) (bool, error)` since shard is derived from the ID (and hence less "important" in the list of arguments).

Reply:
Sure, I changed the signature